src/build/buildExecutor.ts

```diff
@@ -38,15 +46,19 @@ export class BuildExecutor {
         let outputPath = path.join(input.localRepositoryPath, OUTPUT_FOLDER_NAME);
         fs.emptyDirSync(outputPath);
 
-        let envs = {
+        let envs: any = {
+            'DOCFX_CORRELATION_ID': correlationId,

```

Comment:
Do you need changes in docfx to support more common properties? I recall there is a need to tell if docfx is running local build or not

Reply:
Yeah, you are right, I will update spec too. how about calling it `IsLocalBuild`? and use the command line parameter `--local-build`?

Reply:
In the originally design, the telemetry is a dictionary that allows you to add custom dimensions to docfx. Let me change docfx first, but you are good with this change.

Reply:
Cool, I will add a TODO on the extension side